data/json/items/chemicals_and_resources.json

```diff
@@ -890,5 +890,24 @@
     "weight": 212,
     "volume": 1,
     "to_hit": -2
-    }
+    },{
+    "type" : "COMESTIBLE",
+    "id" : "sewage_water",
```

Comment:
Should be `water_sewage` to match `water_clean`, `water_acid`, `water_mineral` etc.